src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -99,6 +104,17 @@ public static Builder builder(Class<?> type) {
     return builder(ClassName.get(type));
   }
 
+  public static AnnotationSpec get(AnnotationMirror annotation) {
```

Comment:
Reading your first comment once more - you didn't voted against the get():AnnotationSpec signature, right? Just the location within the file was the issue.